helm/ffc-grants-eligibility-checker/templates/config-map.yaml

```diff
@@ -1,16 +1,16 @@
 {{- include "ffc-helm-library.container-config-map" (list . "ffc-grants-eligibility-checker.config-map") -}}
 {{- define "ffc-grants-eligibility-checker.config-map" -}}
 data:
-  NODE_ENV: {{ quote .Values.environment }}
-  PORT: {{ quote .Values.container.port }}  
-  SERVICE_BUS_HOST: {{ .Values.container.messageQueueHost | quote }}
-  SCORE_REQUEST_QUEUE_ADDRESS: {{ .Values.container.scoreRequestQueueAddress | quote }}
-  SCORE_RESPONSE_QUEUE_ADDRESS: {{ .Values.container.scoreResponseQueueAddress | quote }}
-  DESIRABILITY_SUBMITTED_TOPIC_ADDRESS: {{ quote .Values.container.desirabilitySubmittedTopicAddress }}
-  NOTIFY_EMAIL_TEMPLATE: {{ .Values.container.notifyEmailTemplate | quote }}
+  NODE_ENV: {{ quote .Values.environment }} 
   SESSION_CACHE_TTL: {{ .Values.container.sessionCacheTTL | quote }}  
-  USE_REDIS: {{ .Values.container.useRedis | quote }}
+  USE_REDIS: {{ quote .Values.container.useRedis | quote }}
```

Comment:
Why duplicating `quote` here?

Reply:
My autocomplete did that without noticing, as it did to SESSION_CACHE_TTL which is why we had the previous error. Edited.